frontend/src/bundles/common/components/create-avatar/components/dots.tsx

```diff
@@ -0,0 +1,90 @@
+import { Circle } from '../../components.js';
```

Comment:
Same here

Reply:
I was thinking at least two levels weren't that bad, but no problem, I'll update it